src/test/resources/com/puppycrawl/tools/checkstyle/checks/coding/illegaltype/InputIllegalTypeMemberModifiers.java

```diff
@@ -3,7 +3,7 @@
 import java.util.Hashtable;
 //configuration: default
 public class InputIllegalTypeMemberModifiers {
-    private AbstractClass a = null; //WARNING
+    private com.puppycrawl.tools.checkstyle.checks.coding.illegaltype.InputIllegalTypeMemberModifiers.AbstractClass a = null; //WARNING
```

Comment:
what is this?

Reply:
I'll remove this. This shouldn't be here. It is a change is from my other issue. IntelliJ must have added it while smart checkout.

Reply:
Done.